apps/core/declarativewebcontainer.cpp

```diff
@@ -865,7 +865,7 @@ void DeclarativeWebContainer::componentComplete()
 
 void DeclarativeWebContainer::updateContentOrientation(Qt::ScreenOrientation orientation)
 {
-    if (gForceLandscapeToPortrait) {
+    if (gForceLandscapeToPortrait || QGuiApplication::primaryScreen()->primaryOrientation() == Qt::LandscapeOrientation) {
```

Comment:
Is there some reason not to use screen()->primaryOrientation()?

Reply:
No reason, did not realise the webcontainer was a window. Fixed. :slightly_smiling_face: